Mathlib/Topology/Compactness/Compact.lean

```diff
@@ -935,6 +935,11 @@ theorem ClosedEmbedding.isCompact_preimage {f : X → Y} (hf : ClosedEmbedding f
   hf.toInducing.isCompact_preimage (hf.closed_range) hK
 #align closed_embedding.is_compact_preimage ClosedEmbedding.isCompact_preimage
 
+/-- The preimage of a compact set in the image of an open embedding is compact. -/
+lemma OpenEmbedding.isCompact_preimage {f : X → Y} (hf : OpenEmbedding f) {K : Set Y}
+    (hK: IsCompact K) (Kf : K ⊆ range f) : IsCompact (f ⁻¹' K) := by
+  rwa [hf.toInducing.isCompact_iff, image_preimage_eq_of_subset Kf]
```

Comment:
Why not generalize this to `hf : Inducing f`?

Reply:
Generalised and upgraded. (I tried that, but got stuck for some reason.)
And yes, I saw the other version already. :-)